src/main/java/com/nike/cerberus/domain/input/VpcAccessWhitelist.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2016 Nike, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nike.cerberus.domain.input;
+
+import java.util.List;
+
+/**
+ * Stores CIDRs to whitelist for VPC access

```

Comment:
There are similar one-liners to describe the rest of the Java Beans.

Any objections? @tlisonbee @fieldju